module/x/gravity/types/genesis.go

```diff
@@ -35,7 +35,7 @@ var (
 	// ParamsStoreKeySignedSignerSetTxsWindow stores the signed blocks window
 	ParamsStoreKeySignedSignerSetTxsWindow = []byte("SignedSignerSetTxWindow")
```

Comment:
I just noticed this and I'm not sure if it matters, but worth checking into -- the param here is `SignedSignerSetTxsWindow` but we're using the byte string `SignerSignerSetTxWindow` (no plural on Tx).